plugins/discordexperiments.plugin.js

```diff
@@ -2,27 +2,30 @@
  * @name discordExperiments
  * @description Enables the experiments tab in discord's settings.
  * @author square
- * @version 1.3.0
+ * @version 1.3.1
  * @website https://betterdiscord.app/plugin/Discord%20Experiments
  * @source https://github.com/Inve1951/BetterDiscordStuff/blob/master/plugins/discordexperiments.plugin.js
  * @updateUrl https://betterdiscord.app/gh-redirect?id=206
  */
 
-const storeExports = BdApi.findModule(m => Reflect.has(m?.default, "isDeveloper"));
-const original = storeExports.default;
+const settingsStore = BdApi.findModule(m => typeof m?.default?.isDeveloper !== "undefined");
+const userStore = BdApi.findModule(m => m?.default?.getUsers);
 
 module.exports = class {
 	getName(){ return "Discord Experiments"; }
 
 	start() {
-    storeExports.default = new Proxy(original, {
-			get(_, key) {
-				return key === "isDeveloper" ? true : original[key];
-			}
-		});
+		const nodes = Object.values(settingsStore.default._dispatcher._dependencyGraph.nodes);
```

Comment:
```js
const nodes = Object.values(settingsStore.default._dispatcher._actionHandlers._dependencyGraph.nodes);
```

Reply:
![DiscordPTB_0075086092](https://user-images.githubusercontent.com/7843719/184241334-a750515b-3ef8-4032-8666-bbce8ea6d036.png)

Still does not work

Reply:
@dav1312 Pull request #189 combines your change.